src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -44,18 +46,20 @@ def get_snuba_query_string(snuba_query: SnubaQuery) -> str:
         SnubaQueryEventType.EventType.DEFAULT: "default",
```

Comment:
not part of this change set, but might be nice to move this static variable out of the method scope so it's not redefined each method invocation.

Reply:
I moved it out 👍